.github/workflows/key4hep.yml

```
@@ -28,6 +28,7 @@ jobs:
           mkdir build install
           cd build
           cmake -DENABLE_SIO=ON \
+            -DENABLE_JULIA=ON \
```

```suggestion
```
This stack doesn't come with julia, so no need to run it here
```suggestion
            -DENABLE_JULIA=OFF \
```

---

Wait, let me make all these changes that you suggested.